source/panel_predictor/__main__.py

```diff
@@ -13,12 +13,11 @@ def display_frame(frame, target=None):
     cv2.imshow('Press q to quit', frame)
```

Comment:
Please remove "from source.panel_predictor.panel_predictor import PanelPredictor" from the top of each main file